src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+      onsubmit="onSubmit()">
+      <div>
+        <fieldset>
+          <legend>Personal information</legend>
+          <div class="entra">
+            <label for="surna">
+              Surname:
+              <input
+                type="text"
+                name="surname"
+                id="surna"
+                maxlength="10">
+            </label>
+          </div>
+          <div class="entra">
+            <label for="names">
+              Name: <input type="text" name="name"
+              id="names" maxlength="10">
+            </label>
```

Comment:
Fix **ALL** the inputs in your project.
```suggestion
              Name:
              <input 
                type="text"
                name="name"
                id="names"
                maxlength="10"
              >
            </label>
```